multicluster/cmd/gateways.go

```diff
@@ -127,19 +130,34 @@ func newGatewaysCommand() *cobra.Command {
 					if !isTargetClusterMetric(metrics, gateway.clusterName) {
 						continue
 					}
-					gatewayStatus.latency = uint64(metrics.GetGauge().GetValue())
+					gatewayStatus.Latency = uint64(metrics.GetGauge().GetValue())
 					break
 				}
 
 				statuses = append(statuses, gatewayStatus)
 			}
-			renderGateways(statuses, stdout)
+
+			switch opts.output {
+			case "":
+				renderGateways(statuses, stdout)
+			case "json":
+				out, err := json.MarshalIndent(statuses, "", "  ")
+				if err != nil {
+					fmt.Fprint(os.Stderr, err)
+					os.Exit(1)
+				}
+				fmt.Printf("%s", out)
+			default:
+				return errors.New("output must be one of: json")
```

Comment:
The error statement seems to be a bit confusing. `output must be one of` should be followed by at least 2 choices. We only have one option i.e `json`.